src/numJewelsInStones.js

```diff
@@ -28,7 +28,17 @@
  * @return {number}
  */
 function numJewelsInStones(jewels, stones) {
-  // write code here
+  const stonesClean = stones.replace(/[^a-zA-Z ]/gi, '');
+  if (jewels === '' || stonesClean === '') {
```

Comment:
Добавь отступ после создания переменных
Пустая строка в булевом значении `false` 
```suggestion
  if (!jewels || !stonesClean) {
```